src/foam/comics/v2/DAOBrowserView.js

```diff
@@ -102,6 +102,7 @@ foam.CLASS({
         return this.DAOControllerConfig.create({ dao: this.data });
       }
     },
+    'columns',
```

Comment:
I think this should be part of the config. Make it a StringArray and call it defaultColumns. Use the model's tableColumns to prefill this.

Reply:
But columns only really make sense depending on the view right? Like as the config doesn't make sense when its rendering a TreeView.

Reply:
Yeah TreeView doesn't get the config. This view does and already makes decisions based on it. I'm not sure why this would be any different.

Reply:
Done.